src/tradingkit/data/feed/backtest_feeder.py

```diff
@@ -50,4 +50,5 @@ def dispatch_month(self, exchange, symbol, year, month, since, to):
 
         for trade in json.load(open(full_filename, 'r')):
             if since.timestamp() <= trade['timestamp'] / 1000 < to.timestamp():
+                trade['exchange'] = self.exchange
```

Comment:
self.exchange is a object or string?

event data must be serializable

Reply:
is a string